automation/release-bumper/release-bumper.sh

```
@@ -4,6 +4,8 @@ set -ex
 
 CONFIG_FILE="hack/config"
 
+source "hack/validate-kv-fg-file.sh"
```

Shellcheck (0.11.0)

[info] Not following: hack/validate-kv-fg-file.sh was not specified as input (see shellcheck -x).

(SC1091)